candies/cli/parsers/standard.py

```diff
@@ -76,15 +77,57 @@ def configure(parser: ArgumentParser, command: Command, prefix: str = '.'):
         {'.command': 'tool', '..command': 'install', 'g': 'something'}.
     """
 
+    def is_(x: inspect.Parameter, of: type) -> bool:
+        return isinstance(x, of) or x == of
+
+    def configure_arg(x: inspect.Parameter, config: Dict):
+        parser.add_argument(parameter.name,
+                            **config)
+
+    def configure_flag(x: inspect.Parameter, config: Dict):
+        # This actually makes an arg to behave like a flag,
+        # so one could call `--x` instead of `--x y`.
+        config['action'] = 'store_true'
+
+        if parameter.default is parameter.empty:
+            config['default'] = False
+        else:
+            config['default'] = bool(parameter.default)
+
+        short = getattr(parameter.annotation, 'short', None)
+        short = short if short is not None else parameter.name[0]
+
+        if short is '':
+            raise ValueError(f"Short name of the flag '{parameter.name}' "
+                             f"is too short")
+
+        if len(short) > 1:
```

Comment:
I’m pretty sure we need to check whether short name is actually a letter.

Reply:
Do we perform any checks for duplicate short names?

Reply:
They're done automatically by `argparse`.